app/src/main/java/org/simple/clinic/di/DateFormatterModule.kt

```diff
@@ -2,35 +2,65 @@ package org.simple.clinic.di
 
 import dagger.Module
 import dagger.Provides
+import org.simple.clinic.appconfig.Country
 import org.simple.clinic.di.DateFormatter.Type.FileDateTime
+import org.threeten.extra.chrono.EthiopicChronology
+import java.time.chrono.Chronology
+import java.time.chrono.IsoChronology
 import java.time.format.DateTimeFormatter
 import java.util.Locale
 import javax.inject.Named
 
 @Module
 class DateFormatterModule {
 
+  @Provides
+  fun providesChronology(country: Country): Chronology {
+    return when (country.isoCountryCode) {
+      Country.ETHIOPIA -> EthiopicChronology.INSTANCE
+      else -> IsoChronology.INSTANCE
+    }
```

Comment:
Should we move this `when` clause to a computed property for the `Country` type, similar to what we have done for the `alternativeIdentifierType`?

Reply:
Yeah, good point. Will move it inside `Country` as a computed property.